typescript/src/actions/solana/transfer.ts

```diff
@@ -7,14 +7,13 @@ import {
 } from "@solana/spl-token";
 import {
   Connection,
-  MessageV0,
   PublicKey,
   SystemProgram,
-  TransactionInstruction,
-  TransactionMessage,
-  VersionedTransaction,
+  SYSVAR_RECENT_BLOCKHASHES_PUBKEY,
+  Transaction,
```

Comment:
Isn't VersionedTransaction the newer transaction type that people are using on Solana?